From: Iain Lane <iainl@gnome.org>
Date: Mon, 22 Jul 2019 21:58:21 +0100
Subject: cogl/tests: Only install run-tests.sh when building installed tests

This is a script for installed tests; leave it out otherwise.

https://gitlab.gnome.org/GNOME/mutter/merge_requests/694

Origin: upstream, commit:59fb26cb00546f2ab5bd7b643e2dfd77d613b2f3
Applied-Upstream: 3.33.90
---
 cogl/tests/meson.build | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/cogl/tests/meson.build b/cogl/tests/meson.build
index 4bb1864..0d76c8b 100644
--- a/cogl/tests/meson.build
+++ b/cogl/tests/meson.build
@@ -6,7 +6,9 @@ cdata.set('HAVE_GLES2', have_gles2.to_int())
 
 cogl_installed_tests_libexecdir = join_paths(
   mutter_installed_tests_libexecdir, 'cogl', 'conform')
-install_data('run-tests.sh', install_dir: cogl_installed_tests_libexecdir)
+if have_installed_tests
+  install_data('run-tests.sh', install_dir: cogl_installed_tests_libexecdir)
+endif
 
 cogl_config_env = configure_file(
   input: 'config.env.in',
